tensorflow/python/ops/histogram_ops.py

```diff
@@ -119,6 +119,11 @@ def histogram_fixed_width(values,
   Returns:
     A 1-D `Tensor` holding histogram of values.
 
+  Raises:
+    TypeError: If any unsupported dtype is provided.
+    errors_impl.InvalidArgumentError: If value_range does not
```

Comment:
no reference to internal symbols, just say tf.InvalidArgumentError

Reply:
@alextp Thanks. The PR has been updated.